app/server/lib/SQLiteDB.ts

```diff
@@ -262,6 +265,13 @@ export class SQLiteDB implements ISQLiteDB {
     return this._db.interrupt?.();
   }
 
+  public backup(filename: string): Backup {
+    if (!this._db.backup) {
+      throw new Error('SQLite wrapper does not support backups');
+    }
+    return this._db.backup?.(filename);
```

Comment:
```suggestion
    return this._db.backup(filename);
```

I don't have the code pulled down to check, but I think the earlier condition should narrow the type here.